src/pluginplay/printing/mermaid.cpp

```diff
@@ -14,7 +14,13 @@
  * limitations under the License.
  */
 
+#include "module_manager/module_manager_class.hpp"
 #include <pluginplay/printing/mermaid.hpp>
-#include <string>
+#include <pluginplay/pluginplay.hpp>
```

Comment:
```suggestion
```
This header is meant only for users of PluginPlay and should never be used in the `include/` or `src/` directories. Long story short, it pulls in the entire library which will increase compilation times for the library and create unnecessary couplings.